src/containers/AccountConnection.jsx

```diff
@@ -128,6 +129,7 @@ class AccountConnection extends Component {
           <KonnectorInstall
             account={createdAccount}
             connector={konnector}
+            onCancel={onCancel}
```

Comment:
It is written in the commit "Make it work" but I see no tests changed/created. Are there tests testing the correct behavior ?

Reply:
You're right there's no test. Since there's no test at all for this flow, I would need some time to write it.

Reply:
Actually this was added to handle cancelling the intent when clicking on the "quit" button of an old cozy-keys-lib's `UnlockForm` design. This button doesn't exist anymore, so this is useless now..

Thus, we can add a test that checks we correctly render a TriggerManager with `vaultClosable=false` prop, ensuring we can't close the `VaultUnlocker` without providing a good password. Would it be OK for you for this case?

Reply:
I don't understand perfectly the code here, I think you are the best to judge whether a test is needed here.

Reply:
See https://github.com/cozy/cozy-home/pull/1433/commits/a40c2ec59be51a4d3911e05dd62378255e42556e